### ISSUE TITLE
Add accuracy score

### ISSUE THREAD

**Issue by nya-elimu** [elimu-ai/ml-storybook-reading-level#20]
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html
